main.py

```diff
@@ -0,0 +1,8 @@
+from parse_input import parse_args
+from commands.command_factory import CommandFactory
+
+if __name__ == "__main__":
+   args = parse_args()
+   args.command = input("Enter create: ")
```

Comment:
I don't see why we need this, args.command already has value which is the command.
And what about other commands like get or set, you say "Enter create" but what if I want to enter another command?

Reply:
+1